src/widget/wtracktableview.cpp

```diff
@@ -549,6 +549,11 @@ void WTrackTableView::onSearch(const QString& text) {
 void WTrackTableView::onShow() {
 }
 
+void WTrackTableView::mousePressEvent(QMouseEvent* pEvent) {
+    m_mousePressPosition = pEvent->position();
```

Comment:
position() is Qt6 only. For my understanding we like to keep 2.5 build-able to check for Qt6 regressions. 
localPos( )is the Qt5 equivalent